components/Sidebar/Sidebar.tsx

```diff
@@ -60,7 +58,7 @@ const Sidebar: FC = () => {
 
       return (
         <li key={i}>
-          <Link
+          <a
```

Comment:
Any specific reason for not using Next's Link component?

Reply:
I was debugging of what could be the reason of not getting redirected. 
Like if I click on saved, it should get redirected but it didn't get. So, I checked by using `<a href=""`

Reply:
It was working fine before when I was working on this component, but the new page may take time to load in development mode. Could you confirm if it's still not working with Next's Link component?

Reply:
Not working when using Next's Link